pkg/flowaggregator/flowaggregator.go

```diff
@@ -381,6 +383,7 @@ func (fa *flowAggregator) Run(stopCh <-chan struct{}, wg *sync.WaitGroup) {
 
 func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
 	expireTimer := time.NewTimer(fa.activeFlowRecordTimeout)
+	logTicker := time.NewTicker(time.Minute)
```

Comment:
Any reason for one minute? Or just going for an arbitrary time period?
Do you plan to use these logs for the flow aggregator restart test?